nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/pom.xml

```diff
@@ -21,6 +21,8 @@ language governing permissions and limitations under the License. -->
 
     <properties>
         <amqp-client.version>5.8.0</amqp-client.version>
+        <common-text.version>1.8</common-text.version>
```

Comment:
Is common-text is used somewhere? I don't find it.

Reply:
> Hey @sedadgn! Thank you for implementing the alternative approach. I've found few things, can you check them? None of them is a big thing so I also think we are really close to merging. One slight thing. There are a lot of inconsistently placed empty lines, missing spaces. They are not considered blocker issues in the community but can I ask you to clean them? I'd appreciate if you could do.

@nandorsoma 
Thank you for your review. I wrote few comment. I made few changing according your comment. 
Best Regards